smartsim/_core/launcher/dragon/dragonBackend.py

```diff
@@ -539,21 +549,83 @@ def _stop_steps(self) -> None:
                 self._group_infos[step_id].status = SmartSimStatus.STATUS_CANCELLED
                 self._group_infos[step_id].return_codes = [-9]
 
-    @property
-    def infra_ddict(self) -> str:
-        """Create a Dragon distributed dictionary and return its
-        serialized descriptor
+    def _create_backbone(self) -> BackboneFeatureStore:
+        """
+        Creates a BackboneFeatureStore if one does not exist. Updates
+        environment variables of this process to include the backbone
+        descriptor.
+
+        :returns: The descriptor of the backbone feature store
```

Comment:
It seems to return an object of type `BackboneFeatureStore`, not its descriptor. Are they the same thing?

Reply:
nope. copypasta fail. fixed